gcloud-java-bigquery/src/main/java/com/google/gcloud/spi/DefaultBigQueryRpc.java

```diff
@@ -91,7 +92,14 @@ private static BigQueryException translate(IOException exception) {
 
   private static BigQueryException translate(GoogleJsonError exception) {
     boolean retryable = RETRYABLE_CODES.contains(exception.getCode());
-    return new BigQueryException(exception.getCode(), exception.getMessage(), retryable);
+    BigQueryError bigqueryError = null;
+    if (exception.getErrors() != null  && !exception.getErrors().isEmpty()) {
+      GoogleJsonError.ErrorInfo error = exception.getErrors().get(0);
```

Comment:
Do you know whether it's possible to get multiple errors in the same `GoogleJsonError` (and if so, would it be helpful to return a list of errors)?  I realize I also only [take the first error](https://github.com/GoogleCloudPlatform/gcloud-java/blob/master/gcloud-java-resourcemanager/src/main/java/com/google/gcloud/spi/DefaultResourceManagerRpc.java#L60) in Resource Manager as well